lib/node_modules/@stdlib/array/empty-like/docs/types/index.d.ts

```diff
@@ -742,10 +51,10 @@ declare function emptyLike( x: Uint8ClampedArray, dtype?: DataType ): Uint8Clamp
 * var arr = emptyLike( x );
 * // returns [ 0.0, 0.0 ]
 */
-declare function emptyLike( x: Array<any>, dtype?: DataType ): Array<number>;
+declare function emptyLike<T extends AnyArray>( x: T ): T;
```

Comment:
I don't think this is correct for "generic" arrays. IIRC, when `x` is a "generic" array, an `Array<number>` is always returned, as the return array is zero-filled. For typed arrays, this type definition is correct. Generic arrays are an exception.

Reply:
Good catch! Addressed.